pkg/state/state_nv_ipam_cni.go

```diff
@@ -71,7 +71,7 @@ type NVIPAMManifestRenderData struct {
 //
 //nolint:dupl
 func (s *stateNVIPAMCNI) Sync(
-	ctx context.Context, customResource interface{}, infoCatalog InfoCatalog) (SyncState, error) {
+	ctx context.Context, customResource any, infoCatalog InfoCatalog) (SyncState, error) {
```

Comment:
As we use `interface{}` over all our codebase I prefer to use the same approach across all the code so we don't need this change